subprojects/docs/src/docs/userguide/userguide_single.adoc

```diff
@@ -23,195 +23,243 @@
 :meta-name-twitter_url: {docsUrl}/{gradleVersion}/userguide/{docname}.html
 :meta-name-twitter_image: {website}/images/gradle-256x256.png
 
-toc::[]
+toc::[leveloffset=+2]
 
 [[part:about_gradle]]
-== About Gradle
+== **OVERVIEW**
 
 include::userguide.adoc[leveloffset=+2]
+include::about_manual.adoc[leveloffset=+2]
 
-[[part:getting_started]]
-== Getting Started
-
-include::introduction.adoc[leveloffset=2]
-
-include::installation.adoc[leveloffset=+2]
+'''
 
-include::troubleshooting.adoc[leveloffset=+2]
+[[part:releases]]
+== **RELEASES**
 
 include::compatibility.adoc[leveloffset=+2]
+include::feature_lifecycle.adoc[leveloffset=+2]
 
 [[part:upgrading_and_migrating]]
-== Upgrading and Migrating
+== UPGRADING
 
+include::upgrading_version_8.adoc[leveloffset=+2]
+include::upgrading_version_7.adoc[leveloffset=+2]
 include::upgrading_version_6.adoc[leveloffset=+2]
-
 include::upgrading_version_5.adoc[leveloffset=+2]
-
 include::upgrading_version_4.adoc[leveloffset=+2]
 
-include::migrating_from_maven.adoc[leveloffset=+2]
+== MIGRATING
 
+include::migrating_from_maven.adoc[leveloffset=+2]
 include::migrating_from_ant.adoc[leveloffset=+2]
 
-[[part:running_builds]]
-== Running Gradle Builds
+[[part:getting_started]]
+== **GETTING STARTED**
```

Comment:
💭 Should getting started come first, ahead of even upgrading/migrating?

Reply:
I'm mirroring the current user manual exactly.